tests/_utils/model.js

```diff
@@ -365,3 +379,35 @@ function convertToModelText( withAttributes = false ) {
 		evt.stop();
 	};
 }
+
+// Typecast string value to the original type.
+// 		`'true'` => `true`
+// 		`'1'` => `1`
+//		`'{"x":1,"y":2}'` => `{ x: 1, y: 2 }`
```

Comment:
You could add a sample with a string which can not be parsed and explain why you return an attribute in case of error. "Typecast string value to the original type." is not clear enought.